pkg/config.go

```diff
@@ -0,0 +1,36 @@
+package pkg
+
+
+// this object should be responsible for
+// loading file from external resources /xml/json/yaml
+type RulesContainer struct {
+	ruleMap map[string]map[string]int
+	timing map[string]int
```

Comment:
is `timing` used somewhere? it looks like n

Reply:
forgot to delete it from old implementation.